lib/psulib_traject/workers/hourly_indexer.rb

```diff
@@ -4,15 +4,40 @@ module PsulibTraject
   module Workers
     class HourlyIndexer < Base
       def perform
-        target = Dir.glob("#{ConfigSettings.symphony_data_path}/#{ConfigSettings.symphony_hourlies_subdir}/**/*.m*rc")
+        target = Dir.glob(
+          "#{ConfigSettings.symphony_data_path}/#{ConfigSettings.symphony_hourlies_subdir}/**/*.m*rc"
```

Comment:
I would do something like: 

``` ruby
def hourlies_directory
  @hourlies_directory ||= Pathname
                            .new(ConfigSettings.symphony_data_path)
                            .join(ConfigSettings.symphony_hourlies_subdir)
end
```

Then here do
``` ruby
target_files = hourlies_directory
  .children
  .select { |filename| filename.extname.match?(/ma?rc/' }

Reply:
oh, i didn't realize that .children would do the same thing as `/**/*` thanks.